src/table/components/PaginationContent.tsx

```diff
@@ -143,7 +155,7 @@ function PaginationContent({
 
   const pageOptions = (
     <>
-      {[...Array(totalPages).keys()].map((pageIdx, index) => (
+      {Array.from(Array(totalPages).keys()).map((pageIdx, index) => (
```

Comment:
ts was giving me hell here. The error message told me do add a flag to the tsconfig. I did and the error was still there. This is basically the same to me so I didn't bother digging deeper

Reply:
Mario might be able to help, although not sure if he has access to this repo,

Reply:
I do not see any error or warning here. Sometimes rebooting VScode helps.

Reply:
with the old implementation I mean, with Array.from() there is no error